Tests/EndToEndTests/EvalClientTests/CNTKLibraryCSEvalExamplesTest/MemoryTests.cs

```diff
@@ -60,5 +62,67 @@ public static void WriteOutputs()
             }
             Console.WriteLine("\nAll saved object references are printed.");
         }
+
```

Comment:
Please add a comment that this test will be moved to the C# unit test project when it is ready.